mission_control/test/test_jaus_ros_bridge_interface_mission_control.py

```diff
@@ -121,59 +124,55 @@ def test_jaus_ros_bridge_interface(self):
         self.mission_to_load.mission_file_full_path = "NO_MISSION"
         self.simulated_mission_control_load_mission_pub.publish(
             self.mission_to_load)
+
         def load_wrong_mission():
             return self.mission_load_state == ReportLoadMissionState.FAILED
         self.assertTrue(
             wait_for(load_wrong_mission),
             msg='Mission control must report FAILED')
 
         # A valid mission is sent to the mission control
-        self.mission_to_load.mission_file_full_path = self.dir_path + \
-            "test_missions/mission.xml"
-        self.simulated_mission_control_load_mission_pub.publish(
-            self.mission_to_load)
-        def load_valid_mission():
-            return self.mission_load_state == ReportLoadMissionState.SUCCESS
-        self.assertTrue(
-            wait_for(load_valid_mission),
-            msg='Mission control must report SUCCESS')
-
         # Execute Mission and check if the mission control reports status
-        mission_to_execute = ExecuteMission()
-        mission_to_execute.mission_id = 1
-        self.simulated_mission_control_execute_mission_pub.publish(mission_to_execute)
-        def mission_status_is_reported():
-            return ReportExecuteMissionState.EXECUTING in self.report_execute_mission.execute_mission_state
+        self.mission.load_mission("mission.xml")
+        self.mission.execute_mission()
+
+        # Wait for the mission to report EXECUTING
+        def executing_mission_status_is_reported():
+            return (ReportExecuteMissionState.EXECUTING in self.mission.execute_mission_state)
```

Comment:
@patapufate nit: no need for parenthesis

Reply:
Fixed